src/Init/Data/BitVec/Basic.lean

```diff
@@ -669,4 +669,11 @@ def ofBoolListLE : (bs : List Bool) → BitVec bs.length
 | [] => 0#0
 | b :: bs => concat (ofBoolListLE bs) b
 
+/- ### reverse -/
+
+/-- Reverse of a Bitvec. We treat BitVec as lists of bools. -/
```

Comment:
```suggestion
/-- Reverse the bits in a bitvector. -/
```

We generally phrase a docstring for a function as an action/verb rather than just the result/noun.

I also wouldn't mention that we treat a bitvector as a list of bools, that's the "expected" interpretation of a bitvector (list of bools is more or less the same as a "vector of bits").

I guess you're referring here to the fact that the implementation is more in the style of a `List.reverse` versus other definitions that rely on the Nat-based internal encoding. That kind of comment feels more appropriate as a local comment rather than a docstring, but even then, I'm not sure it needs mentioning; it's clear from the code already.